ui/src/utils/constants.js

```diff
@@ -204,14 +204,114 @@ export const PROVIDER_DEFAULT_CONFIGS = new Map([
     'google',
     {
       minCapacity: 0,
-      maxCapacity: 0,
-      capacityPerInstance: 1,
-      machineType: 'n1-highcpu-8',
-      regions: ['us-west2'],
-      workerConfig: {},
-      scheduling: {},
-      networkInterfaces: [{}],
-      disks: [{}],
+      maxCapacity: 4,
+      launchConfigs: [
+        {
+          region: 'us-west1',
+          zone: 'us-west1-a',
+          capacityPerInstance: 1,
+          disks: [
+            {
+              autoDelete: true,
+              boot: true,
+              initializeParams: {
+                diskSizeGb: 50,
+                sourceImage:
+                  'projects/taskcluster-imaging/global/images/docker-worker-gcp-googlecompute-2019-09-25t19-53-35z',
```

Comment:
we can replace this with `...`

Reply:
Hi @imbstack will I be replacing just the object or the `initializeParams` key as well?

Thanks in advance!